### PR TITLE
updates to /resources tree

### PR DESCRIPTION
**Pull request by dr-wesley**

### CODE REVIEW

File: docs/resources/core/console_history.md [hunk 1]
```diff
@@ -1,29 +1,33 @@
- # oci\_core\_console_history
+# oci\_core\_console_history
+
+[ConsoleHistory Reference][03f57b12]
+
+  [03f57b12]: https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/20160918/ConsoleHistory/ "ConsoleHistoryReference"
```
Comment (codycushing) [inline docs/resources/core/console_history.md]: Hi Dan, I'm unfamiliar with this linking structure, can you describe how it works?

**Comment by dr-wesley** [comment]
Hey Cody,

Let’s talk tomorrow, I’ve been sick all week and will be wfh, but we can talk

dan

File: docs/resources/objectstorage/object.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # oci\_objectstorage\_bucket
 
 Provides an Objectstorage resource for CRUD operations on objects.
+`**Dan: not sure which links to use here**`
```
Comment (briangustafson) [inline docs/resources/objectstorage/object.md]: remove

**Comment by dr-wesley** [comment]
refreshed several files for consistency. Have questions highlighted in the following:
  /resources/objectstorage/object.md
 /resources/core/private_ip.md
 /resources/core/volume.md
Once these are done, should be good to go.

File: docs/resources/core/volume.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
 # oci\_core\_volumes
 
+[Volume Reference][ce7191fd]
+
+  [ce7191fd]: https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/20160918/Volume/ "VolumeReference"
+
 Gets a list of volumes in a compartment.
+`**Dan_should this be providing a resource?**`
```
Comment (briangustafson) [inline docs/resources/core/volume.md]: Yes - data sources list, but resources offer CRUD (or at least CRD).

File: docs/resources/objectstorage/object.md (hunk 2 above)
Comment (briangustafson) [reply docs/resources/objectstorage/object.md]: In answer to the question: There isn't an obvious choice here, as there is with most resources. I would go with https://docs.us-phoenix-1.oraclecloud.com/api/#/en/objectstorage/20160918/Object/GetObject.

File: docs/resources/core/private_ip.md [hunk 4]
```diff
@@ -1,5 +1,11 @@
 # oci\_core\_private_ip
 
+[PrivateIp Reference][b236f15a]
+
+  [b236f15a]: https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/20160918/PrivateIp/ "PrivateIpReference"
+
+**dan: does this get IP info, or does it provide a private IP resource/**
```
Comment (briangustafson) [inline docs/resources/core/private_ip.md]: Provides a resource. PrivateIP is a first class object with its own CRUD operations.

Comment (dr-wesley) [reply docs/resources/core/private_ip.md]: done

**Comment by dr-wesley** [comment]
@codycushing @briangustafson 
pls take a quick peek at my changes and then we can close this one.
thanks
dan

File: docs/resources/core/internet_gateway.md [hunk 5]
```diff
@@ -1,6 +1,10 @@
 # oci\_core\_internet\_gateways
 
-Gets a list of internet gateways.
+[InternetGateway Reference][0162d0a8]
+
+  [0162d0a8]: https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/20160918/InternetGateway/ "InternetGatewayReference"
+
+Gets a list of Internet Gateways.
```
Comment (briangustafson) [inline docs/resources/core/internet_gateway.md]: I know that isn't part of your change, but any resource doc that says "Gets a list of" is wrong - probably copied from the data source.

**Pull request merged**